deep-learning/_posts/2018-08-02-neural-trees.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: review
+title:  "Adaptive Neural Trees"
+tags:   deep-learning CNN  machine-learning classification
+author: Sarah Leclerc
+pdf:   https://arxiv.org/pdf/1807.06699.pdf
+cite:
+  authors: "Tanno, Ryutaro and Arulkumaran, Kai and Alexander, Daniel and Criminisi, Antonio and Nori, Aditya"
+  title:   "Adaptive Neural Trees"
+  venue:   "July 2018"
```

Comment:
please give the venue (conference name, journal name, or if not published yet, the Arxiv ref.)